src/scaleio.rs

```diff
@@ -1584,23 +1596,29 @@ impl Scaleio {
         Ok(instance_statistics)
     }
 
+    //ids is (device_id, sds_id, storage_pool_id)
     pub fn get_drive_statistics(
         &self,
         t: DateTime<Utc>,
-        device_id: &str,
+        ids: &DriveId,
     ) -> MetricsResult<Vec<TsPoint>> {
         let instance_statistics = get::<DeviceStatistics>(
             &self.client,
             &self.config,
-            &format!("instances/Device::{}/relationships/Statistics", device_id),
+            &format!("instances/Device::{}/relationships/Statistics", ids.id),
         )
         .and_then(|instance| {
             let points: Vec<TsPoint> = instance
                 .into_point(Some("scaleio_drive_stat"), true)
                 .iter_mut()
                 .map(|point| {
                     point.timestamp = Some(t);
-                    point.add_tag("device_id", TsValue::String(device_id.to_string()));
+                    point.add_tag("device_id", TsValue::String(ids.id.to_string()));
```

Comment:
I bet we could eliminate the clone here.
Would something like this compile?
```
instance.into_point().into_iter()/*take ownership*/.map().collect()
```
I bet that'd elminate the `point.clone()` on 1622 and maybe even the `to_string()` calls

Reply:
hmmm.  Lemme try

Reply:
![image](https://user-images.githubusercontent.com/9067743/67584836-770d0e00-f71c-11e9-9f2c-96d4114faab6.png)

Reply:
unless that's not what you wanted

Reply:
![image](https://user-images.githubusercontent.com/9067743/67585865-65c50100-f71e-11e9-9b64-228db3bf93d4.png)

Reply:
![image](https://user-images.githubusercontent.com/9067743/67586024-b4729b00-f71e-11e9-9dda-8024743a11ca.png)
new problem